relays/table_transform.ts

```diff
@@ -0,0 +1,43 @@
+"use strict";
+
+import { Message } from "mhub";
+
+/**
+ * @param msg Message just received on this binding.
+ * @return Message(s), promise for message(s), or void
+ */
+let ROUND = "Round";
+let NUMBER = "No.";
+let NAME = "Name";
+let TOP = "Highest";
+
+function roundsArray(rounds: number): string[] {
+	let arr = [];
+	for(let i = 0; i < rounds; i++) {
+		arr.push(ROUND + ' ' + i);
+	}
+	return arr;
+}
+
+export default function(msg: Message): void|Message|Message[]|Promise<Message> {
+	if (msg.topic === "scores:ranking") {
+		let result = msg.clone();
+		let old_data = result.data;
+		result.data = {};
+
+		let rounds = roundsArray(old_data.stage.rounds);
+		result.data.header = [NUMBER, NAME, TOP].concat(rounds);
+		result.data.data = old_data.ranking.map(function(rank) {

```

Comment:
You could opt for replacing `function(rank) { ...}` with the more modern `(rank) => { ...}` style.